daq/runner.py

```diff
@@ -1075,6 +1078,13 @@ def _send_device_result(self, device, test_results):
         except Exception as e:
             LOGGER.error("Failed to send device results for device %s: %s ", device, e)
 
+    def report_sink(self, report_dict):
```

Comment:
If it's not consumed by some grpc client, does it really have to be in the protobuf? The existing dict / a dataclass to replace https://github.com/faucetsdn/daq/blob/master/daq/report.py#L240 should suffice.